internal/spdxlicense/generate_license_list.go

```diff
@@ -1,3 +1,4 @@
+//go:build ignore
```

Comment:
Just curious — what is this for?

Reply:
removed - my local golang configuration updated this